.travis.yml

```diff
@@ -2,4 +2,5 @@ language: node_js
 node_js:
   - lts/dubnium
   - node
+before_script: npm install katex
```

Comment:
katex will be installed without extra install scripts because it is already in devdeps.